src/backend/chat/collate.py

```diff
@@ -3,78 +3,130 @@
 
 from backend.model_deployments.base import BaseDeployment
 
+RELEVANCE_THRESHOLD = 0.5
+
 
 def combine_documents(
```

Comment:
this function just calls and returns `rerank_and_chunk` now. seems redundant to wrap it, maybe we can just call `rerank_and_chunk` directly now?